test.sh

```diff
@@ -1,29 +1,32 @@
 #!/usr/bin/env bash
 set -e
 
-while getopts t:r:b: flag
+while getopts t:r:b:c: flag
 do
     case "${flag}" in
         t) test=${OPTARG};;
         r) runs=${OPTARG};;
         b) build=${OPTARG};;
+        c) config=${OPTARG};;
     esac
 done
 
 runs=$([ -z "$runs" ] && echo "10" || echo "$runs")
 build=$([ -z "$build" ] && echo "1" || echo "$build")
+config=$([ -z "$config" ] && echo "config.json" || echo "$config")
 skip_build=$([ "$build" == "0" ] && echo "1" || echo "0")
 
 [[ $SKIP_MAINNET_CHECK || "$ETH_RPC_URL" && "$(seth chain)" == "ethlive" ]] || { echo "Please set a mainnet ETH_RPC_URL"; exit 1; }
 
-export DAPP_TEST_TIMESTAMP=1615792486
-export DAPP_TEST_NUMBER=12045000
+export DAPP_TEST_TIMESTAMP=1622483493
+export DAPP_TEST_NUMBER=12543537
 export DAPP_SOLC_VERSION=0.6.11
 export DAPP_SRC="contracts"
-export DAPP_BUILD_OPTIMIZE=1
+export DAPP_BUILD_OPTIMIZE=0
 export DAPP_BUILD_OPTIMIZE_RUNS=200
-export DAPP_LINK_TEST_LIBRARIES=1
+export DAPP_LINK_TEST_LIBRARIES=0
 export DAPP_REMAPPINGS=$(cat remappings.txt)
+export DAPP_LIBRARIES=" contracts/libraries/loan/v1/LoanLib.sol:LoanLib:0x51A189ccD2eB5e1168DdcA7e59F7c8f39AA52232 contracts/libraries/pool/v1/PoolLib.sol:PoolLib:0x2c1C30fb8cC313Ef3cfd2E2bBf2da88AdD902C30 contracts/libraries/util/v1/Util.sol:Util:0x95f9676A34aF2675B63948dDba8F8c798741A52a"
```

Comment:
we can re-use the build.sh here again to avoid confusion.

Reply:
I wanted to keep them separate so we can have a separate job in CI to run the build